src/common/utils.py

```diff
@@ -177,10 +183,39 @@ def get_s3_file_md5(bucket_name, key):
 """
 create error dict
 """
-def create_error(title, msg):
-    return {"title": title, "description": msg}
+def create_error(code, msg, property_name, property_value):
+    message_config = load_message_config()
```

Comment:
Can we move this loading configuration out to run only once when service starts? It seems it will reload the config every time an error needs to be generated.

Reply:
It will only load once.     The key is if not hasattr(load_message_config, 'message_config'):   So if the function has no the attribute of 'message_config', it will load.  otherwise it return the attribute.

Reply:
I see!